docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -275,11 +275,25 @@ $ rhoas context use --name development-context
 $ cd ~/app-services-guides/code-examples/quarkus-service-registry-quickstart/
 ----
 
+. Generate an environment variables file that contains the credentials for service account.
++
+[source,shell]
+----
+$ rhoas service-account create --type env --output-file ./producer/.env
+----
+
 . Generate an environment variables file that contains the connection configuration information required by the producer component.
 +
 [source,shell]
 ----
-$ rhoas generate-config --type env --output-file ./producer/.env
+$ rhoas generate-config --type env --output-file ./producer/rhoas.env
+----
+
+. Append the contents of connection configuration file to the service account environment variables file.
++
+[source,shell]
+----
+$ cat ./producer/rhoas.env >> ./producer/.env
```

Comment:
Does this appending approach look good? I am not sure if quarkus can load `rhoas.env`, hence appended URLs to service account file.
@wtrocki